mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientidentifiers/ClientIdentifiersFragment.java

```diff
@@ -130,16 +132,34 @@ public void showUserInterface() {
     public void showClientIdentifiers(List<Identifier> identifiers) {
         this.identifiers = identifiers;
         identifierListAdapter.setIdentifiers(identifiers);
+        identifierListAdapter.notifyDataSetChanged();
```

Comment:
I agree with you ! ```DiffUtils``` is not relevant for our case. However I just wanted to you to know about it :) It is optimized for large operations in our case we won't have very large lists to play with. You can even copy new lists to old ones :P 
However since we know about additions and deletions we can use more precise methods than just ``` notifyDataSetChanged()```

Makes sense ?